vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -75,7 +75,8 @@ def validate_pages(cls, pages):
     def validate_navigation(cls, navigation, values):
         if "pages" not in values:
             return navigation
-        if navigation is None:
+
+        if navigation is None or not navigation.pages:
```

Comment:
Ah yes thank you! 👍 All good.

FYI in case I forget again... the reason the second of those cases might be useful is so you could do something like this in the future:
```
dashboard.navigation = Navigation(selector=NavBar())  # change selector type but otherwise leave navigation the same
```
and then it would work without the user needing to explicitly list all the `pages` themselves. It's nice if a user can incrementally build and alter their navigation like this, as they can for a `Filter`.